trino/dbapi.py

```diff
@@ -23,16 +23,9 @@
 
 from typing import Any, List, Optional  # NOQA for mypy types
 
-try:
-    from urllib.parse import urlencode
-except ImportError:
-    # Python 2
-    from urllib import urlencode
```

Comment:
Was this unused?

This drops Python 2 support. The sad truth is that many people are still on 2.7.

Reply:
@hashhar per the diff, looks like unused import?

Reply:
It was unused.